File: School/src/edu/northeaststate/cs1/Labs/Lab03/Lab03.java
/**
 * ---------------------------------
 * File Name: Lab03.java
 * Project Name: Lab03
 * ---------------------------------
 * Creator's Name: Tyler Burleson
 * Email: tburles6@stumail.northeaststate.edu
 * Course: CISP 1010 TR
 *Creation Date: 9/2/2021
 *----------------------------------
 */
public class Lab03 {
    public static void main(String[] args) {

        System.out.println("▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐\n" +
                "▐     Tyler Burleson-CISP 1010       ▐\n" +
                "▐         Lab    03                  ▐\n" +
                "▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐▐");

        // declares and assigns our variables to be used in the printed algorithm
        double velocity0 = 5.6;
        double velocity1 = 10.5;
        double time = 0.5;
        double acceleration = (velocity1 - velocity0)/ time;

        // prints variables
        System.out.println("velocity0 = " + velocity0 + "m/s");
        System.out.println("velocity1 = " + velocity1 + "m/s");
        System.out.println("time = " + time + "s");
        System.out.println("The acceleration is: " + acceleration + "m/s^2");
    }
}

File: School/src/edu/northeaststate/PHP/Labs/Lab03/Lab03.php
<!--
File: Lab03-php.php
Project: Lab 3
Creator: Tyler Burleson
Email: tburles6@stumail.northeaststate.edu
Course: CITC 1317 TR
Creation Date: 9/21/21
-->
<?php
$lines = file('80s_vg_sales.csv');
$video_games = array();
?>

<?php
foreach ($lines as $line) {
    $video_games[] = explode(",", trim($line));


}
?>



<html>
<head>
    <title>Lab03</title>
</head>
<body>
<h2>80's Game Sales</h2>

<?php
echo "<table border ='1'>";
foreach ($video_games as $game){
    echo "<tr>";
    echo "<td>" . $game[0] . "</td>";
    echo "<td>" . $game[1] . "</td>";
    echo "<td>" . $game[2] . "</td>";
    echo "<td>" . $game[3] . "</td>";
    echo "<td>" . $game[4] . "</td>";
    echo "<td>" . $game[5] . "</td>";
    echo "</tr>\n";

}
echo "</table>";
$total = 0;
$firstIteration = true;
foreach($video_games as $game){
    if(!$firstIteration){
        $total += $game[5];
    }
    $firstIteration = false;
}
$total = $total * 100000;
echo "Total Estimated Global Sales: $total";

?>



</body>
</html>